tests/v2/validation/charts/cisbenchmark_test.go

```diff
@@ -0,0 +1,298 @@
+	suite.Suite
+	cisProfileName      string
+	client              *rancher.Client
+	session             *session.Session
+	project             *management.Project
+	chartInstallOptions *charts.InstallOptions
+}
+
+func (c *CisBenchmarkTestSuite) TearDownSuite() {
+	c.session.Cleanup()
+}
+
+func (c *CisBenchmarkTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	c.session = testSession
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(c.T(), err)
+
+	c.client = client
+
+	cisConfig := new(CisConfig)
+	config.LoadConfig(CisConfigFileKey, cisConfig)
+	c.cisProfileName = cisConfig.ProfileName
+	require.NotEmptyf(c.T(), c.cisProfileName, "CIS profile name required")
+
+	clusterName := client.RancherConfig.ClusterName
+	require.NotEmptyf(c.T(), clusterName, "Cluster name to install is not set")
+
+	clusterID, err := clusters.GetClusterIDByName(client, clusterName)
+	require.NoError(c.T(), err)
+
+	if cisConfig.ChartVersion == "" {
+		cisConfig.ChartVersion, err = client.Catalog.GetLatestChartVersion(charts.RancherCisBenchmarkName, catalog.RancherChartRepo)
+		require.NoError(c.T(), err)
+	}
+
+	projectConfig := &management.Project{
+		ClusterID: clusterID,
+		Name:      charts.CisbenchmarkProjectName,
+	}
+	createdProject, err := client.Management.Project.Create(projectConfig)
+	require.NoError(c.T(), err)
+	require.Equal(c.T(), createdProject.Name, charts.CisbenchmarkProjectName)
+	c.project = createdProject
+
+	c.chartInstallOptions = &charts.InstallOptions{
+		ClusterName: clusterName,
+		ClusterID:   clusterID,
+		Version:     cisConfig.ChartVersion,
+		ProjectID:   createdProject.ID,
+	}
+
+}
+
+func (c *CisBenchmarkTestSuite) TestCisBenchmarkChart() {
+	subSession := c.session.NewSession()
+	defer subSession.Cleanup()
+
+	client, err := c.client.WithSession(subSession)
+	require.NoError(c.T(), err)
+
+	c.T().Log("Installing latest version of cis benchmark chart")
+	err = charts.InstallRancherCisBenchmarkChart(client, c.chartInstallOptions)
+	require.NoError(c.T(), err)
+
+	c.T().Log("Waiting for cisbenchmark chart deployments to have expected number of available replicas")
+	err = charts.WatchAndWaitDeployments(client, c.project.ClusterID, charts.RancherCisBenchmarkNamespace, metav1.ListOptions{})
+	require.NoError(c.T(), err)
+
+	scanName := namegen.AppendRandomString("scan")
+
+	c.T().Log("Running a scan")
+	yamlInput := &management.ImportClusterYamlInput{
+		DefaultNamespace: charts.RancherCisBenchmarkNamespace,
+		YAML:             fmt.Sprintf(ClusterScanYaml, scanName, c.cisProfileName),
+	}
+
+	cluster, err := client.Management.Cluster.ByID(c.project.ClusterID)
+	require.NoError(c.T(), err)
+	_, err = client.Management.Cluster.ActionImportYaml(cluster, yamlInput)
+	require.NoError(c.T(), err)
+
+	steveClient, err := client.Steve.ProxyDownstream(c.project.ClusterID)
+	require.NoError(c.T(), err)
+
+	err = wait.Poll(1*time.Second, 5*time.Minute, func() (done bool, err error) {
```

Comment:
please use the default times we have defined now when possible

Reply:
Thanks! Updated the code